src/screens/CreatePost/elements/Card.js

```diff
@@ -1,51 +1,52 @@
 import React from 'react';
-import {View, StyleSheet, Platform} from 'react-native';
-import {COLORS} from '../../../utils/theme';
-
-const Card = (props) => {
+import { View, StyleSheet, Platform } from 'react-native';
+import { colors } from '../../../utils/colors';
+const Card = props => {
   return (
     // <View style={{ ...styles.card, ...props.style }}></View>
     <View // Parent
       style={{
         // No backgroundColor
-        shadowColor: COLORS.black,
-        shadowOffset: {width: 0, height: 1},
+        shadowColor: '#000',
+        shadowOffset: { width: 0, height: 1 },
         shadowOpacity: 0.5,
         // shadowRadius: 10,
         zIndex: 999,
         // width: widthInput,
-        ...props.style
-      }}>
+        ...props.style,
+      }}
+    >
       <View // Card
         style={{
           flex: 1,
           borderRadius: 10,
           // To round image corners
           overflow: 'hidden',
-          borderColor: COLORS.silver,
+          borderColor: '#bdbdbd',
           borderWidth: 0.5,
           // https://github.com/facebook/react-native/issues/10049#issuecomment-366426897
-          backgroundColor: COLORS.lightgrey,
+          backgroundColor: colors.lightgrey,
           // Android shadow
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `backgroundColor` property has been changed from `COLORS.lightgrey` to `colors.lightgrey`. Ensure that `colors.lightgrey` is correctly defined in the new color scheme.

```diff
- backgroundColor: COLORS.lightgrey,
+ backgroundColor: colors.lightgrey,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->